pulseapi/settings.py

```diff
@@ -13,6 +13,19 @@
 import os
 import dj_database_url
 
+import environ
+
+app = environ.Path(__file__) - 1
+root = app - 1
+env = environ.Env(
```

Comment:
Because I just copied this from the science api and have no idea what I'm actually doing yet.